annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -261,8 +252,9 @@ protected ResponseEntity<String> storeAnnotations(String annotationPageIn, Authe
 	    apRes.setTotalInPage(resList.size());
 //			apRes.setCurrentPageUri("http://UNDEFINED");
 
-	    String jsonLd = (new AnnotationPageSerializer(apRes)).serialize(SearchProfiles.STANDARD);
-
+	    annotationPageSerializer.setProtocolPage(apRes);
```

Comment:
the serializer is statefull, therefore new object must be created for each request